packages/components/src/text-control/stories/index.tsx

```diff
@@ -0,0 +1,65 @@
+/**
+ * External dependencies
+ */
+import type { ComponentMeta, ComponentStory } from '@storybook/react';
+
+/**
+ * WordPress dependencies
+ */
+import { useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import TextControl from '..';
+
+const meta: ComponentMeta< typeof TextControl > = {
+	component: TextControl,
+	title: 'Components/TextControl',
+	argTypes: {
+		onChange: {
+			action: 'onChange',
+			control: { type: null },
```

Comment:
Nit: It looks like we could remove the `control: { type: null }` in this case. Storybook hides the controls for `on*` props by default.

Reply:
Thanks. I didn't know that 👍 Removed in [e30a8dd](https://github.com/WordPress/gutenberg/pull/40633/commits/e30a8ddf2e7abf6548bf3b71d0d4df7ddf66f280)